libethereum/EthereumHost.cpp

```diff
@@ -120,21 +120,6 @@ void EthereumHost::doWork()
 		foreachPeer([](std::shared_ptr<EthereumPeer> _p) { _p->tick(); return true; });
 	}
 
-	if (m_syncStart)
```

Comment:
It was like that:
1. sync() tries to find PV63 peer.
2. If failed, it sets the m_syncStart and does not create BlockChainSync.
3. In the reset function here, if 10s after m_syncStart the BlockChainSync is created.

I just tried to create the BlockChainSync straight away it I can't see any difference. Maybe it had more sense when there was PV61/62 support.